packages/client/hmi-client/src/types/SimulateConfig.ts

```diff
@@ -15,6 +15,18 @@ export type ChartConfig = {
 	selectedRun: string;
 };
 
+export type CalibrateConfig = {
+	runConfigs: { [runId: string]: any };
+	chartConfigs: string[][];
+};
+
+export type CalibrateStore = {
+	runId: string;
+	active: boolean;
+	loss: { [key: string]: number }[];
+	params: { [key: string]: number };
+};
+
```

Comment:
I don't quite understand this - did we get the files mixed up? This is `simulate` but you are adding `calibrate` related definitions, also "CalibrateStore" doesn't seem to be used.

Reply:
moved to its own file now